pandas/tseries/tests/test_period.py

```diff
@@ -4075,8 +4075,10 @@ def test_pi_ops_nat(self):
                                 'NaT', '2011-06'], freq='M', name='idx')
         self._check(idx, lambda x: x + 2, expected)
         self._check(idx, lambda x: 2 + x, expected)
+        self._check(idx, lambda x: np.add(x, 2), expected)
```

Comment:
Added a test case. Because current ``.values`` is internally `int``, it doesn't raise error as expected. Because #13988 will change ``.values``, let me revisit this at that time.